guides/modeling-electricity.md

```diff
@@ -0,0 +1,50 @@
+# Modeling Electricity
+
+Specifying the electrical service on a wire or circuit warrants a bit of explanation. This guide shows the simple way to specify the electrical characteristics, 
+and then goes into more detail about what is "under the hood" interms of modeling it.
+
+## Using predefined electrical service
+The 223 standard comes with a predefined vocabulary of common electrical service, so the easiest way to model is simply to pick the appropriate service from the controlled vocabulary. 
+Shown below is a small excerpt of available AC electrical service.
+
+![ACServiceExcerpt](images/guides-AC-service-excerpt.png)
+
+There is a naming convention to help the human reader in understanding each option, but note that later in this guide we will show how all the 
+properties of the service are queryable without resorting to parsing the name of the service. With that disclaimer, 
+a human reader can interpret the service s223:AC-120VLN-1Ph-60Hz to mean "AC electricity, carrying 120 volts between Line and Neutral, single-phase, at 60Hz frequency".
+
+A more complex example is s223:AC-240VLL-208VLN-120VLN-3Ph-60Hz, meaning "three-phase, 60Hz AC electricity, with a 240V line-line voltage, a 208V line-neutral voltage and also a 127V line-neutral voltage".
+
+Assuming your desired electrical service is listed in the vocabulary, the simplest way to model the electricity in the building is to use the s223:hasMedium relation, as shown below:
+
+![ACServiceExample](images/guides-AC-service-example.png)
+
+## High-Leg-Delta Distribution Panel
+Here's a diagram of a "High-Leg-Delta" distribution panel that provides several different voltages and phases.
+
+...
+
+...and here's the model:
```

Comment:
```suggestion
```

Reply:
[sample_highLegDelta_electrical_entry.zip](https://github.com/open223/docs.open223.info/files/14070784/sample_highLegDelta_electrical_entry.zip)

The model of HighLegDelta from Bob